### DISCUSSION TITLE
Tauri react desktop app disable pinch zoom

### DISCUSSION THREAD

**Discussion by IlimaVIP** [tauri-apps/tauri#8643] [Q&A]
I have a desktop app running on Linux touch screen desktop. I need to allow users to interact with app using touch screen but disable zooming in to the app and zooming out, I’ve tried with meta for html and also tried with css to disable pinch zooming, but still let’s to pinch zoom, how can I disable in web view for Tauri?

**Comment by edmundoferreira** [comment]
@IlimaVIP i believe your question is similar to https://github.com/tauri-apps/tauri/discussions/3843
